_posts/2023-07-31-paris-dev-sprint.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "scikit-learn 2023 In-person Developer Sprint in Paris, France"
+date: July 31, 2023
+
+categories:
+  - Events
+tags:
+  - Sprints
+  - Community
+
+postauthors:
+  - name: Reshama Shaikh
+    website: https://reshamas.github.io
+    image: reshama_shaikh.jpeg 
+  - name: François Goupil
+    email: francois.goupil@inria.fr
+    website: https://github.com/francoisgoupil
+    image: "francois_goupil.jpeg"
+---
+<div>
+  {% include postauthor.html %}
+</div>
+
+
+In June of 2023 (week of June 19 to 23), the scikit-learn team held its first developers sprint since 2019! The sprint took place in Paris, France at the Dataiku office. The sprint event was an in-person event only and had 29 participants.
+
+The following [scikit-learn team members](https://scikit-learn.org/stable/about.html) joined the sprint:  
+
+1. Gael Varoquaux (@GaelVaroquaux)
+1. Olivier Grisel (@ogrisel)
+1. Guillaume Lemaitre (@glemaitre)
+1. Tim Head (@betatim)
+1. Maren Westermann
+1. Arturo Amor Quiroz
+1. Adrin Jalali
+1. Loïc Estève
+1. Thomas Fan
+1. Julien Jerphanion (@jjerphan)
+1. François Goupil (@francoisgoupil)
+1. Roman Yurchak
+1. Joris Van den Bossche
+
+The following community members joined the sprint:  
+
+1. Ines (intern at Dataiku)
+1. Chaine San Buenaventura
+1. Yann Lechelle
+1. Vincent Maladière
+1. Franck Charras
+1. Jovan Stojanovic
+1. Leo Dreyfus-Schmidt
+1. Alexandre Landeau
+1. Louis Fouquet
+1. Alexandre Vigny
+1. Harizo Rajaona
+1. Samuel Ronsin
+1. Riccardo Cappuzzo
+1. Camille Troillard
+1. Denis Engemann
+
+
+<figure>
+ <img src="/assets/images/posts_images/2023-paris-sprint/paris_2023.jpg" alt="group of people who participated in the sprint" max-width="20%" max-height="20%" /> 
+ <figcaption>
+ scikit-learn Developer Sprint, Paris, June 2023; Photo credit: <a href=" "> Inria / Photo B. Fourrier, June 2023</a>; (from left to right, back to front): XXX, XXX, François Goupil, Tim Head, Guillaume Lemaitre, XXX, XXX, Gael Varoquaux, Olivier Grisel, Alexandre Gramfort [add other people in]
+ </figcaption>
+</figure>
+
+## Sponsors
+- Dataiku provided the space and some of the food, as well as all of the coffee.
+- The scikit-learn consortium paid for the lunch, the travel and accommodation expenses.
```

Comment:
I would mention that the scikit-learn consortium organized the sprint and paid for the lunch...